templates/components/common/footer.html

```diff
@@ -54,15 +54,17 @@ <h5 class="footer-info-heading">{{lang 'footer.brands'}}</h5>
                     <li><a href="{{urls.brands}}">{{lang 'common.view_all'}}</a></li>
                 </ul>
             </article>
-            {{/if}}
+            {{/and}}
 
+            {{#if settings.address}}
```

Comment:
if you enter phone number it will require you to enter address as well. No way to have only phone number.